packages/commonwealth/shared/schemas/createCommunitySchema.ts

```diff
@@ -40,7 +40,8 @@ export const createCommunitySchema = z.object({
   icon_url: z
     .string()
     .url()
-    .superRefine(async (val, ctx) => await checkIconSize(val, ctx)),
+    .superRefine(async (val, ctx) => await checkIconSize(val, ctx))
+    .optional(),
```

Comment:
@jnaviask in the new create community flow icon url should be mandatory per product requirement